linux/files/90autoupdates

```diff
@@ -0,0 +1,29 @@
+{%- set autoupdates = salt['pillar.get']('linux:system:autoupdates') %}
+
+{%- if autoupdates.enabled %}
```

Comment:
You are right, I didn't notice you are  already defining params in map.jinja.
Sometimes we are using condition like `if pillar.linux.system.autoupdates is defined` to bypass map.jinja and decide whether to include linux.system.autoupdates. But it's not this used here, then you approach is fine.